clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/CloudDriverConfig.groovy

```diff
@@ -381,11 +382,13 @@ class CloudDriverConfig {
   @Bean
   DescriptionAuthorizerService descriptionAuthorizerService(Registry registry,
                                                             Optional<FiatPermissionEvaluator> fiatPermissionEvaluator,
-                                                            SecurityConfig.OperationsSecurityConfigurationProperties opsSecurityConfigProps) {
+                                                            SecurityConfig.OperationsSecurityConfigurationProperties opsSecurityConfigProps,
```

Comment:
Has spotless run on this? Seems much further indented than normal.

Reply:
Not sure that spotless runs on Groovy files.